src/translations/opl3bankeditor_fr_FR.ts

```diff
@@ -1,6 +1,60 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
 <TS version="2.1" language="fr" sourcelanguage="en">
+<context>
+    <name>AudioConfigDialog</name>
+    <message>
+        <location filename="../audio_config.ui" line="14"/>
+        <source>Audio configuration</source>
+        <translation>Configuration audio</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="20"/>
+        <source>Device</source>
+        <translation>Périphérique</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="28"/>
+        <source>Select the sound device:</source>
+        <translation>Sélectionner le périphérique audio :</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="35"/>
+        <source>...</source>
+        <translation>...</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="80"/>
+        <source>Latency</source>
+        <translation>Latence</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="86"/>
+        <source>Define the audio latency.</source>
+        <translation>Définir la latence audio.</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="127"/>
+        <source>ms</source>
+        <translation>ms</translation>
+    </message>
+    <message>
+        <location filename="../audio_config.ui" line="136"/>
+        <source>If audio glitches occur, you should reset this setting to a higher value.</source>
+        <translation>Si des problèmes audio se produisent, réassignez à ce paramètre à une valeur plus élevée.</translation>
```

Comment:
il semble y avoir un "à" en trop avant "ce paramètre"

Reply:
My bad, you're absolutely right. Corrigé